webcompat/static/js/lib/bugform.js

```diff
@@ -5,9 +5,9 @@
 function BugForm() {
   this.form = $("#js-ReportForm form");
   this.submitButtons = $("#js-ReportForm .js-Button");
-  this.loadingIndicator = $(".js-Loader");
+  this.loadingIndicator = $(".js-loader");
```

Comment:
I'm not sure how useful this is for the non-image upload case. If there's no image upload, how will the user know to look there for opacity changes to indicate progress?

Reply:
> If there's no image upload, how will the user know to look there for opacity changes to indicate progress?

I'm not sure I understand this line? 🤔

Reply:
Right now there are 2 loader elements: loading indicator to indicate the form is being submitted, and loading indicator to indicate an image is being uploaded. You have both of these selectors on the same element, which is the image upload preview area. 

Since the effect of the loader is to apply an animation that "pulses" on opacity, how does the user notice  that if they haven't uploaded an image? Maybe if you upload a screenshot of both loader indicators in action it will make more sense (with and without an image preview).

Reply:
Another thing that doesn't quite make sense with this approach:

https://github.com/webcompat/webcompat.com/blob/7694b0a5c26b08f01a60680d18d6841c28944cca/webcompat/static/js/lib/bugform.js#L89

On form submit we're going to call `show()` on an already visible image upload preview area. Is that intentional? The previous design worked because there was a display:none animated image, which was shown then hidden.

Reply:
> Maybe if you upload a screenshot of both loader indicators in action it will make more sense (with and without an image preview).

Now I get it. Yes, agreed. Lets give it a try. 

Just some context regarding the spinner / message etc.: 
As we talked about the loader we really wanted to have something that is not blocking the UI what would happen with an overlay. This is why I'd rather try to find a different solution than that.